assets/offering/anmal_skador.svg

```diff
@@ -0,0 +1,33 @@
+<svg width="270" height="258" viewBox="0 0 270 258" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
can we be more consistent in naming? Why are we a) naming this in swedish b) naming it in snake case

Reply:
good point, did not change from the download, and i think the majority are actually in swedish. Maybe save for later and make a separate ticket for all images to english and update all usages?

Reply:
sure, but plz change to kebab case :)